### PR TITLE
Support for Rails 7.2

### PR DESCRIPTION
**Pull request by adbelsham**
Adds support for Rails 7.2.

### CODE REVIEW

**Comment by mathieujobin** [comment]
Can you please update the Changelog and bump the version as well?

**Comment by mathieujobin** [comment]
looks like 7.2 has to exclude ruby 3.0
unless we remove support for ruby 3.0 entirely as well ?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,10 +15,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rails: [7.1.1, 7.0.8, 6.1.7]
+        rails: [7.2.0, 7.1.1, 7.0.8, 6.1.7]
         ruby: [3.3, 3.2, 3.1, 3.0.6, jruby-9.4]
```
Comment (mathieujobin) [inline .github/workflows/build.yml:19]: I think removing 3.0.6 from this line will suffice to move us forward... 
not sure if we should add an explicit include for 3.0.x below or just ignore?

**Comment by adbelsham** [comment]
> Can you please update the Changelog and bump the version as well?

This is done now.

**Comment by adbelsham** [comment]
> looks like 7.2 has to exclude ruby 3.0 unless we remove support for ruby 3.0 entirely as well ?

I've added an exclude line, please approve the workflows to test this has resolved the problems.

File: .github/workflows/build.yml (hunk 1 above)
Comment (adbelsham) [reply .github/workflows/build.yml:19]: I added an explicit exclude in the end.

**Comment by adbelsham** [comment]
Thanks for the review @mathieujobin 
I've actioned your comments, please re-run the tests and let me know if any further changes required.

**Comment by mathieujobin** [comment]
released and pushed to rubygems, thanks for your contribution @adbelsham

**Pull request merged**